2.4/README.md

```diff
@@ -25,6 +25,9 @@ The following environment variables influence the MongoDB configuration file. Th
 |  `MONGODB_NOPREALLOC` | Disable data file preallocation.                                          |  true
 |  `MONGODB_SMALLFILES` | Set MongoDB to use a smaller default data file size.                      |  true
 |  `MONGODB_QUIET`      | Runs MongoDB in a quiet mode that attempts to limit the amount of output. |  true
+|  `MONGODB_TEXT_SEARCH_ENABLED` | Enables the [text search](https://docs.mongodb.org/v2.4/core/index-text/#text-search-text-command) feature.  |  false
+
+**Notice: The `MONGODB_TEXT_SEARCH_ENABLED` option is deprecated in MongoDB 2.6.**
```

Comment:
@bparees first off, this option is enabled by default in mongo 2.6 which will change our default to "true" there and cause inconsistency between image functionality....  after talking to @omron93 I would rather opt-out this option for 2.6 and wait if somebody requests disabling.